### PR TITLE
Bump `py-ssz` to `0.1.3` and update the fuzzing tests

### PR DESCRIPTION
**Pull request by hwwhww**
* Bump `py-ssz` to `0.1.3`
* Enable the `spec.BeaconState` & `spec.BeaconBlock` in fuzzing test case.
* Enable py-ssz hash_tree_root assertions

### CODE REVIEW

**Comment by djrtwo** [comment]
If this is relevant to fuzzing, should we release soon via `v08x`?

**Comment by hwwhww** [comment]
The `test_libs/pyspec/eth2spec/fuzzing/` seems only testing against `py-ssz` and `pyspec`, no test cases output right now. I think both `dev` and `v08x` are fine, your call. :)

File: test_libs/pyspec/eth2spec/fuzzing/test_decoder.py [hunk 1]
```diff
@@ -9,9 +9,7 @@ def test_decoder():
     rng = Random(123)
 
     # check these types only, Block covers a lot of operation types already.
-    # TODO: Once has Bitlists and Bitvectors, add back
-    #       spec.BeaconState and spec.BeaconBlock
-    for typ in [spec.IndexedAttestation, spec.AttestationDataAndCustodyBit]:
+    for typ in [spec.BeaconState, spec.BeaconBlock]:
```
Comment (protolambda) [inline test_libs/pyspec/eth2spec/fuzzing/test_decoder.py]:
```suggestion
    for typ in [spec.IndexedAttestation, spec.AttestationDataAndCustodyBit, spec.BeaconState, spec.BeaconBlock]:
```

File: test_libs/pyspec/eth2spec/fuzzing/decoder.py [hunk 2]
```diff
@@ -19,13 +19,13 @@ def translate_typ(typ) -> ssz.BaseSedes:
...
     elif issubclass(typ, spec_ssz.List):
         # TODO: Make py-ssz List support the new fixed length list
-        return ssz.List(translate_typ(typ.elem_type))
+        return ssz.List(translate_typ(typ.elem_type), typ.length)
     elif issubclass(typ, spec_ssz.Bitlist):
         # TODO: Once Bitlist implemented in py-ssz, use appropriate type
-        return ssz.List(translate_typ(typ.elem_type))
+        return ssz.Bitlist(typ.length)
     elif issubclass(typ, spec_ssz.Bitvector):
         # TODO: Once Bitvector implemented in py-ssz, use appropriate type
```
Comment (protolambda) [inline test_libs/pyspec/eth2spec/fuzzing/decoder.py]:
```suggestion
```

File: test_libs/pyspec/eth2spec/fuzzing/decoder.py [hunk 3]
```diff
@@ -19,13 +19,13 @@ def translate_typ(typ) -> ssz.BaseSedes:
         return ssz.Vector(translate_typ(typ.elem_type), typ.length)
     elif issubclass(typ, spec_ssz.List):
         # TODO: Make py-ssz List support the new fixed length list
-        return ssz.List(translate_typ(typ.elem_type))
+        return ssz.List(translate_typ(typ.elem_type), typ.length)
     elif issubclass(typ, spec_ssz.Bitlist):
         # TODO: Once Bitlist implemented in py-ssz, use appropriate type
```
Comment (protolambda) [inline test_libs/pyspec/eth2spec/fuzzing/decoder.py]:
```suggestion
```

File: test_libs/pyspec/eth2spec/fuzzing/decoder.py [hunk 4]
```diff
@@ -19,13 +19,13 @@ def translate_typ(typ) -> ssz.BaseSedes:
         return ssz.Vector(translate_typ(typ.elem_type), typ.length)
     elif issubclass(typ, spec_ssz.List):
         # TODO: Make py-ssz List support the new fixed length list
```
Comment (protolambda) [inline test_libs/pyspec/eth2spec/fuzzing/decoder.py]:
```suggestion
```

**Comment by protolambda** [comment]
Added a few suggestions. Please use the batch functionality when applying them.

Also, I think `08x` is a better base for this PR. Can't wait to start fuzzing 0.8 :)

File: test_libs/pyspec/eth2spec/fuzzing/test_decoder.py (hunk 1 above)
Comment (hwwhww) [reply test_libs/pyspec/eth2spec/fuzzing/test_decoder.py]: `IndexedAttestation ` is part of `AttesterSlashing` in `BeaconBlock. attester_slashings`. Will add `AttestationDataAndCustodyBit`!

**Pull request merged**